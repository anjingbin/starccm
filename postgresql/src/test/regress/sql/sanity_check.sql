
VACUUM;
--
-- sanity check, if we don't have indices the test will take years to
-- complete.  But skip TOAST relations since they will have varying
-- names depending on the current OID counter.
--
SELECT relname, relhasindex
   FROM pg_class
   WHERE relhasindex AND relkind != 't'
   ORDER BY relname;

