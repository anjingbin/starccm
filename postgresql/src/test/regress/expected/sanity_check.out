VACUUM;
--
-- sanity check, if we don't have indices the test will take years to
-- complete.  But skip TOAST relations since they will have varying
-- names depending on the current OID counter.
--
SELECT relname, relhasindex
   FROM pg_class
   WHERE relhasindex AND relkind != 't'
   ORDER BY relname;
       relname       | relhasindex 
---------------------+-------------
 bt_f8_heap          | t
 bt_i4_heap          | t
 bt_name_heap        | t
 bt_txt_heap         | t
 fast_emp4000        | t
 func_index_heap     | t
 hash_f8_heap        | t
 hash_i4_heap        | t
 hash_name_heap      | t
 hash_txt_heap       | t
 ihighway            | t
 num_exp_add         | t
 num_exp_div         | t
 num_exp_ln          | t
 num_exp_log10       | t
 num_exp_mul         | t
 num_exp_power_10_ln | t
 num_exp_sqrt        | t
 num_exp_sub         | t
 onek                | t
 onek2               | t
 pg_aggregate        | t
 pg_am               | t
 pg_amop             | t
 pg_amproc           | t
 pg_attrdef          | t
 pg_attribute        | t
 pg_class            | t
 pg_database         | t
 pg_description      | t
 pg_group            | t
 pg_index            | t
 pg_inherits         | t
 pg_language         | t
 pg_largeobject      | t
 pg_opclass          | t
 pg_operator         | t
 pg_proc             | t
 pg_relcheck         | t
 pg_rewrite          | t
 pg_shadow           | t
 pg_statistic        | t
 pg_trigger          | t
 pg_type             | t
 road                | t
 serialtest          | t
 shighway            | t
 tenk1               | t
 tenk2               | t
(49 rows)

